Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -555,30 +567,32 @@ IPCCommandResult ES::DeleteContent(const IOCtlVRequest& request)
 ReturnCode ES::ExportTitleInit(Context& context, u64 title_id, u8* tmd_bytes, u32 tmd_size)
 {
   // No concurrent title import/export is allowed.
-  if (context.title_export.valid)
+  if (context.title_import.valid)
     return ES_EINVAL;
 
   const auto tmd = FindInstalledTMD(title_id);
   if (!tmd.IsValid())
     return FS_ENOENT;
 
-  context.title_export.tmd = tmd;
+  context.title_import = {};
+  context.title_import.tmd = tmd;
 
-  const auto ticket = DiscIO::FindSignedTicket(context.title_export.tmd.GetTitleId());
+  const auto ticket = DiscIO::FindSignedTicket(context.title_import.tmd.GetTitleId());
   if (!ticket.IsValid())
     return ES_NO_TICKET;
-  if (ticket.GetTitleId() != context.title_export.tmd.GetTitleId())
+  if (ticket.GetTitleId() != context.title_import.tmd.GetTitleId())
     return ES_EINVAL;
 
-  context.title_export.title_key = ticket.GetTitleKey(m_ios.GetIOSC());
+  // FIXME: this is wrong. The title key is *not* used here. Key #5 or a null key is.
+  context.title_import.key = ticket.GetTitleKey(m_ios.GetIOSC());
```

Comment:
I assume this will get a follow-up PR or similar...what do you think of a stub implementation (that is currently wrong and contains the FIXME comment) that at least conveys the meaning of what should happen here?